sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -283,8 +283,56 @@ pi_result piDeviceGetInfo(pi_device device, pi_device_info paramName,
     // sycl/doc/extensions/supported/sycl_ext_intel_device_info.md
   case PI_DEVICE_INFO_UUID:
   case PI_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES:
-  case PI_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES:
     return PI_ERROR_INVALID_VALUE;
+  case PI_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES: {
+    // Initialize result to minimum mandated capabilities according to
+    // SYCL2020 4.6.3.2
+    pi_memory_scope_capabilities result = PI_MEMORY_SCOPE_WORK_ITEM |
+                                          PI_MEMORY_SCOPE_SUB_GROUP |
+                                          PI_MEMORY_SCOPE_WORK_GROUP;
```

Comment:
From [Table 21. Context information descriptors](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.context.info):

> `info::context::atomic_memory_scope_capabilities`
> Returns the set of memory scopes supported by atomic operations on all devices in the context, which is guaranteed to include `work_group`.

Why our default return value provides more guarantees than described in a SYCL spec?